rusk-wallet/src/lib/crypto.rs

```diff
@@ -54,18 +61,54 @@ impl MnemSeed {
     }
 }
 
-/// Encrypt the wallet seed
-pub(crate) fn encrypt(
+/// Encrypt the wallet seed using AES-256-CBC
+/// Requires the seed and the encryption key
+/// Will return the ciphertext and the initialization vector (IV)
+pub(crate) fn encrypt_seed(
     seed: &[u8; 64],
     _pwd: String,
-) -> Result<Vec<u8>, CliError> {
-    Ok(seed.to_vec())
+) -> Result<(Vec<u8>, Vec<u8>), CliError> {
+    // this has to be a fresh random value for each execution
+    let iv = rand::thread_rng().gen::<[u8; 16]>();
+
+    let cipher = Aes256Cbc::new_from_slices(_pwd.as_bytes(), &iv).unwrap();
```

Comment:
Note: Hash `pwd` to match N-byte required length (assume random `String` comes in)